qemu/tests/cfg/chardev_acpi.cfg

```diff
@@ -0,0 +1,10 @@
+- chardev_acpi:
+    only x86_64, i386
+    only RHEL
+    vms = 'vm1 vm2'
+    start_vm = no
+    type = chardev_acpi
+    serials += ' vs1 '
```

Comment:
@michiboo  In this case, the two vms do not need boot at the same time. It is my mistake.
If boot two vms with different cfg, we can use
```
serials_vm1 += ' vs1'
extra_params_vm2 = ' -serial /dev/ttyS0'
```
But it is no need. One vm is ok,  boot vm, check and destroy, update params and boot up again.